lib/oelite/fetch/url.py

```diff
@@ -116,34 +117,34 @@ def get_disable_ftp_epsv(self, d):
 
 
 def grab(url, filename, timeout=120, retry=5, proxy=None, ftpmode=False):
-    import urlgrabber
-    import urlgrabber.progress
-    class SimpleProgress(urlgrabber.progress.BaseMeter):
-        def _do_end(self, amount_read, now=None):
-            print "grabbed %d bytes in %.2f seconds" %(amount_read,self.re.elapsed_time())
-
     print "Grabbing", url
     def grab_fail_callback(data):
         # Only print debug here when non fatal retries, debug in other cases
         # is already printed
         if (data.exception.errno in retrycodes) and (data.tries != data.retry):
             print "grabbing retry %d/%d, exception %s"%(
                 data.tries, data.retry, data.exception)
-    try:
-        retrycodes = urlgrabber.grabber.URLGrabberOptions().retrycodes
-        if 12 not in retrycodes:
-            retrycodes.append(12)
-        if not os.path.exists(os.path.dirname(filename)):
-            os.makedirs(os.path.dirname(filename))
-        downloaded_file = urlgrabber.urlgrab(
-            url, filename,timeout=timeout,retry=retry, retrycodes=retrycodes,
-            progress_obj=SimpleProgress(), failure_callback=grab_fail_callback,
-            copy_local=True, proxies=proxy, ftp_disable_epsv=ftpmode)
-        if not downloaded_file:
-            return False
-    except urlgrabber.grabber.URLGrabError as e:
-        warn('URLGrabError %i: %s' % (e.errno, e.strerror))
-        if os.path.exists(filename):
-            os.unlink(filename)
+
+    if not os.path.exists(os.path.dirname(filename)):
+        os.makedirs(os.path.dirname(filename))
+
+    cmd = ['/usr/bin/env', 'wget', '-t', retry, '-T', timeout, '--passive-ftp', '--no-check-certificate', '--progress=dot', '-v', url, '-O', filename]
+
```

Comment:
Why use /usr/bin/env? subprocess.Popen uses execvp, so wget would be search for in $PATH just as env ends up doing - but if for some reason /usr/bin/env doesn't exist, this will fail.